packages/utils/navigation.ts

```diff
@@ -21,6 +21,7 @@ export type RootStackParamList = {
   UserPublicProfile: {
     id: string;
     tab?: keyof typeof uppTabItems;
+    isEditProfile?: boolean;
```

Comment:
Why you add this but you didn't use it in the screen ?
Please remove it if you don't use it